src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -40,25 +41,34 @@ public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData) {
 
         return HearingWindow.builder()
             .firstDateTimeMustBe(getFirstDateTimeMustBe())
-            .dateRangeStart(getHearingWindowStart(caseData))
+            .dateRangeStart(getDateRangeStart(caseData, referenceDataServiceHolder))
             .dateRangeEnd(null)
             .build();
     }
 
+    public static LocalDate getDateRangeStart(
+        @Valid SscsCaseData caseData,
+        ReferenceDataServiceHolder referenceDataServiceHolder
+    ) {
+        return isHearingAdjourned(caseData)
+            && referenceDataServiceHolder.isAdjournmentFlagEnabled() // TODO remove when flag enabled
```

Comment:
(minor) could move the flag to the start for a tiny improvement

Reply:
done